spammer.rb

```diff
@@ -4,30 +4,30 @@
 
 def read_parameters()
   content = JSON.parse(File.read("env.json"))
-  return content["channel"], content["message"], content["sleep_time"].to_i
+  return content[0]["channel"], content[0]["message"], content[0]["sleep_time"].to_i
 end
 
 
 def waiting_send(bot, channel, message, wait)
-  # Wait the time before sending the message
-  puts "I'm now waiting \033[92m#{wait}\033[0m secondes."
-  sleep(wait)
-
-  # Browse the list of server we are
-  bot.servers.each do |serv_id, serv|
+    # Wait the time before sending the message
+    puts "I'm now waiting \033[92m#{wait}\033[0m secondes."
+    sleep(wait)
+    content = JSON.parse(File.read("env.json"))
```

Comment:
why re-reading the config file here ?
and why don't you use de function `read_parameters()` ?